.github/workflows/regression.yml

```diff
@@ -60,10 +60,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix: 
-        testscript: ['tutorials.py', 'parallel_regression.py', 'parallel_regression_AD.py', 'serial_regression.py', 'serial_regression_AD.py', 'hybrid_regression.py', 'hybrid_regression_AD.py']
+        testscript: ['tutorials.py', 'species_regression.py', 'parallel_regression.py', 'parallel_regression_AD.py', 'serial_regression.py', 'serial_regression_AD.py', 'hybrid_regression.py', 'hybrid_regression_AD.py']
         include:
           - testscript: 'tutorials.py'
             tag: MPI
+          - testscript: 'species_regression.py'
+            tag: MPI
```

Comment:
Are you planning to move this to other regression scripts or keep it separate? I can mark it as "required" for the integration tests if you want.

Reply:
usually I port these over to the existing ones at the end of the PR. For development this is really neat because the own reg tests are oc more likely to break and then I see it at a glance. 

We can do either of the possibilities. Currently this split by serial /parallel / hybrid and primal / AD + tutorials gets stuffed more and more and serial/parallel_regression.py are by far the longest running scripts. So maybe just making a serial_regression2.py script and split it in half :D